test/Microsoft.Azure.WebJobs.Host.EndToEndTests/App.config

```diff
@@ -47,12 +47,16 @@
       </dependentAssembly>
       <dependentAssembly>
         <assemblyIdentity name="System.Diagnostics.DiagnosticSource" publicKeyToken="cc7b13ffcd2ddd51" culture="neutral" />
-        <bindingRedirect oldVersion="0.0.0.0-4.0.1.0" newVersion="4.0.1.0" />
+        <bindingRedirect oldVersion="0.0.0.0-4.0.3.0" newVersion="4.0.3.0" />
```

Comment:
Elsewhere the redirect is to `4.0.2.0`... is this one wrong, or are the others?

Reply:
this is a mistake, fixed